src/app/database/migrations/1535181191526-InitialSchema.ts

```diff
@@ -159,6 +162,16 @@ export class InitialSchema1535181191526 implements MigrationInterface {
 				VALUES
 			(1, 1), (2, 2), (3, 2), (4, 3)`
     );
+    await queryRunner.query(
```

Comment:
ahh I am not sure why do we need another set of user_roles?

Reply:
Nice catch. This is just a code smell 💩.